python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -187,6 +187,14 @@
     mysql_length={"asset_key": 64, "partition": 64, "check_name": 64},
 )
 
+db.Index(
+    "idx_asset_check_executions_status",
+    AssetCheckExecutionsTable.c.asset_key,
+    AssetCheckExecutionsTable.c.check_name,
+    AssetCheckExecutionsTable.c.execution_status,
+    mysql_length={"asset_key": 64, "check_name": 64},
+)
```

Comment:
does this need an ordering column as well? how do you imagine we would fetch these in a filtered view? I am guessing theres some dimension for ordering / pagination that we probably need in the index

Reply:
great point